Raspberry Coffee/CoffeeMakerControllerApp/project.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "dependencies": {
+    "Microsoft.ApplicationInsights": "1.0.0",
```

Comment:
If we don't actually care about the telemetry for this sample, remove the ApplicationInsights references everywhere.